recipes/gperftools/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+import os
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+
+
+class GperftoolsConan(ConanFile):
+    name = "gperftools"
+    license = "BSD-3-Clause License"
+    homepage = "https://github.com/gperftools/gperftools"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "gperftools: originally Google Performance Tools"
+    topics = ("malloc", "profiler")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"fPIC": False, "shared": True}
```

Comment:
I am paid by the line count :smile: 
```suggestion
    options = {
        "shared": [True, False],
        "fPIC": [True, False],
    }
    default_options = {
        "shared": True,
        "fPIC": False,
    }
```